product.html

```diff
@@ -0,0 +1,221 @@
+            </div>
+            <div class="insights__details">
+                <img src="./assets/img/product/lead-ins.png" alt="" class="insights__img">
+            </div>
+        </div>
+    </section>
+
+    <section class="insights">
+        <div class="grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10">
+            <div class="insights__details">
+                <img src="./assets/img/product/real-ins.png" alt="" class="insights__img">
+            </div>
+            <div class="insights__details">
+                <div class="insights__gradient-title">Remote teams</div>
+                <div class="insights__title">Real-time collaboration</div>
+                <div class="insights__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.</div>
+            </div>
+        </div>
+    </section>
+
+    <section class="features">
+        <h2 class="features__title">Features</h2>
+        <div class="features__container">
+            <div class="features__item">
+                <img src="./assets/icons/feat1.svg" alt="" class="features__icon">
+                <div class="features__name">Advanced 256-bit encryption</div>
+                <div class="features__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+
+            <div class="features__item features__item--box">
+                <img src="./assets/icons/feat2.svg" alt="" class="features__icon">
+                <div class="features__name">Simple collaboration tools</div>
+                <div class="features__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+
+            <div class="features__item features__item--box">
+                <img src="./assets/icons/feat3.svg" alt="" class="features__icon">
+                <div class="features__name">Customizable AI features</div>
+                <div class="features__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+        </div>
+    </section>
+
+    <div class="view-pricing">
+        <div class="view-pricing__content">
+            <h2 class="view-pricing__title">Are you ready to grow your business with us?</h2>
+            <div class="view-pricing__subtitle">Lorem ipsum dolor sit amet, consetetur
+                Subscribe to our Nesadipscing elitr, sed diam nonumy.</div>
+            <button class="view-pricing__btn">View Pricing</button>
+        </div>
+        <img src="./assets/img/footer-blur-bg.png" alt="" class="view-pricing__blur-bg">
```

Comment:
`view-pricing__blur-bg` is kind of confusing. Try maybe `view-pricing__blurred-background` or similar descriptive names.